CRM/Civicase/Setup/AddManageWorkflowMenu.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use CRM_Civicase_Service_CaseCategoryMenu as CaseCategoryMenu;
+use CRM_Civicase_Service_CaseInstance as CaseInstance;
+
+/**
+ * Adds the Manage Workflow Menu item for existing Case types.
+ */
+class CRM_Civicase_Setup_AddManageWorkflowMenu {
+
+  /**
+   * Updates the Manage Cases Menu URLs.
+   */
+  public function apply() {
+    CaseInstance::assignInstanceForExistingCaseCategories();
```

Comment:
This function has nothing to do `AddManageWorkflowMenu` as it should be possible to use this setup elsewhere and it should do what it says.
Will advice to move this to `CRM_Civicase_Upgrader_Steps_Step0014` and call it just before the CRM_Civicase_Setup_AddManageWorkflowMenu sutup is invoked.

Also this setup is called in the enable function. The `CaseInstance::assignInstanceForExistingCaseCategories();` will not be useful too because at this point no existing case categories exist in the system.